src/assets/locales/fr.json

```diff
@@ -0,0 +1,465 @@
+  "autoexit.title": "La session prendra fin automatiquement dans ",
+  "autoexit.title_units": " secondes",
+  "autoexit.concurrent_subtitle": "Vous avez démarré une nouvelle session.",
+  "autoexit.idle_subtitle": "Vous avez été inactif trop longtemps.",
+  "autoexit.cancel": "ANNULER",
+  "presence.entered_room": "est entré dans la salle.",
+  "presence.entered_lobby": "est arrivé dans le hall.",
+  "presence.join_lobby": "a rejoint le hall.",
+  "presence.join_room": "a rejoint la salle.",
+  "presence.leave": "est parti.",
+  "presence.name_change": "s'appelle à présent",
+  "presence.scene_change": "a choisi la scène",
+  "presence.hub_name_change": "a renommé la salle",
+  "presence.in_lobby": "Hall",
+  "presence.entering": "Arrive dans la salle",
+  "presence.in_room": "Dans la salle",
+  "presence.mute_all": "Couper le son de tous les participants",
+  "home.create_a_room": "Créer une salle",
+  "home.desktop.add_pwa": "Installer une app Desktop",
+  "home.mobile.add_pwa": "Ajouter à l'écran d'accueil",
+  "home.take_a_tour": "Faire une visite",
+  "home.room_create_options": "options",
+  "home.room_create_button": "Créer une salle",
+  "home.create_name.validation_warning": "Nom invalide, limité entre 4 et 64 caractères, symboles restreints.",
+  "home.powered_by_prefix": "Propulsé par",
+  "home.powered_by_link": "Hubs Cloud",
+  "home.join_us": "Rejoignez la conversation",
+  "home.subscribe_to_mailing_list": "Abonnez-vous pour recevoir les dernières nouvelles",
+  "home.have_code": "Vous avez le code d'une salle ?",
+  "home.add_to_discord_1": "Ajouter le",
+  "home.add_to_discord_2": "%app-name% Bot",
+  "home.add_to_discord_3": "à Discord",
+  "home.create_with_spoke": "Créer une scène",
+  "home.report_issue": "Signaler les problèmes",
+  "home.source_link": "Source",
+  "home.whats_new_link": "Actualités",
+  "home.about_link": "A propos",
+  "home.community_link": "Communauté",
+  "home.docs_link": "Documentation",
+  "home.cloud_link": "Hubs Cloud",
+  "home.admin": "Administration",
+  "home.privacy_notice": "Politique de confidentialité",
+  "home.terms_of_use": "Conditions d'utilisation",
+  "home.made_with_love": "fait avec 🦆 par ",
+  "home.environment_author_by": " par ",
+  "home.rooms_title": "Créez des salles en un clic",
+  "home.rooms_blurb": "Partagez des espaces virtuels avec vos amis, vos collègues et vos communautés. Lorsque vous créez une salle avec Hubs, vous pouvez partager sans délai un lien d'invitation vers un espace virtuel privé - aucun téléchargement requis, ni casque de réalité virtuelle.",
+  "home.communicate_title": "Communiquez de manière spontanée",
+  "home.communicate_blurb": "Choisissez un avatar qui vous représente, mettez vos écouteurs et plongez ! Avec Hubs, c'est facile de rester connecté avec d'autres personnes dans une salle privée, en discutant et en chattant.",
```

Comment:
```suggestion
  "home.communicate_blurb": "Choisisser un avatar qui vous représente, metter vos écouteurs et plonger ! Avec Hubs, c'est facile de rester connecté avec d'autres personnes dans une salle privée, en discutant et en chattant.",
```

Reply:
Choisisser > Choisir 
Metter > Mettre

Reply:
We should keep the imperative form, with `ez`, my bad